src/main/java/space/obminyashka/items_exchange/controller/UserController.java

```diff
@@ -83,7 +83,9 @@ public String updateUserInfo(@Valid @RequestBody UserUpdateDto userUpdateDto, @P
             @ApiResponse(responseCode = "404", description = "NOT FOUND")})
     @ResponseStatus(HttpStatus.OK)
     public List<AdvertisementTitleDto> getCreatedAdvertisements(@Parameter(hidden = true) Authentication authentication) {
-        return advService.findAllByUsername(authentication.getName());
+        var username = authentication.getName();
+        var avatarImage = userService.getAvatarByUsername(username);
```

Comment:
That is a mistake and has to be fixed.
The advertisement has to have its title image NOT the user's one.